kodi_addon_checker/check_repo.py

```diff
@@ -24,21 +17,22 @@ def check_repo(repo_path, parameters):
 
     print("Toplevel folders " + str(toplevel_folders))
```

Comment:
We should probably not use print at all anymore, right? So let's use the reporter here also.

Reply:
Yeah.
Shall we completely remove this part because we also have `INFO: checking plugin.foo.bar`?

Reply:
I would like to keep this, this will print all folders it gathered, if we run it on a whole repo.

Reply:
I mean even no need for Reporter at this place.

Reply:
I don't think I get what your saying :/

But can you agree that it's useful information?

Reply:
Removed the line